### PR TITLE
1066 - Add test for delete service action

### PR DESCRIPTION
**Pull request by aldipower**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @Poltergeist

**Comment by Poltergeist** [comment]
Looks good to me.

File: src/js/events/__tests__/MarathonActions-test.js [hunk 1]
```diff
@@ -113,6 +113,57 @@ describe('MarathonActions', function () {
...
+      this.configuration = RequestUtil.json.calls.mostRecent().args[0];
+    });
+
+    it('calls #json from the RequestUtil', function () {
+      expect(RequestUtil.json).toHaveBeenCalled();
+    });
+
+    it('sends data to the correct URL', function () {
+      expect(this.configuration.url)
+        .toEqual(`${Config.rootUrl}/marathon/v2/apps//test`);
```
Comment (orlandohohmeier) [inline src/js/events/__tests__/MarathonActions-test.js]: Do we still need to send the preceding slash?

**Comment by orlandohohmeier** [comment]
@aldipower I'd like to ask you to introduce test like these alongside the actual feature next time (may be even in the same commit)?

**Comment by orlandohohmeier** [comment]
Looks good to me. Let's 🚢  it!

**Comment by aldipower** [comment]
@orlandohohmeier OK, will do it alongside the introduction of the functionality next time.

**Pull request merged**